DS13/code/modules/borg/antag_datum.dm

```diff
@@ -0,0 +1,119 @@
+#define isBorgDrone(D) (D in GLOB.borg_collective.drones)
+
+/datum/antagonist/borg_drone
+	name = "Borg Drone"
+	antagpanel_category = "Borg Drone"
+	job_rank = ROLE_BORG_DRONE
+	show_name_in_check_antagonists = TRUE
+	antag_moodlet = /datum/mood_event/focused
+
+/datum/antagonist/borg_drone/on_gain()
+	. = ..()
+	antag_memory += "We are a <font color='red'><B>borg drone</B></font>. We must assimilate the station at all costs."
+	if(!isBorgDrone(src))
+		GLOB.borg_collective.drones += owner.current //They're in the collective, but need the conversion table for all their upgrades like the tool etc.
+		GLOB.borg_collective.check_completion()
+	var/datum/objective/O = new /datum/objective/survive() //Change this later
```

Comment:
DEEP-SPACE 13 REWORK standardisation

Reply:
I don't understand

Reply:
Commenting as in instead of just //Change this later